### PR TITLE
Logging. Remainder of classes Format specifiers vs concatenation

### PR DESCRIPTION
**Pull request by kingthorin**
Following https://github.com/zaproxy/zaproxy/pull/7451

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
This has conflicts.

**Comment by kingthorin** [comment]
Conflict addresses.

**Comment by kingthorin** [comment]
Conflicts addressed again.

**Comment by kingthorin** [comment]
Got all those.

File: zap/src/main/java/org/zaproxy/zap/authentication/ScriptBasedAuthenticationMethodType.java [hunk 1]
```diff
@@ -518,19 +513,16 @@ private void loadScript(ScriptWrapper scriptW, boolean adaptOldValues) {
                 String[] requiredParams = script.getRequiredParamsNames();
                 String[] optionalParams = script.getOptionalParamsNames();
                 this.loadedCredentialParams = script.getCredentialsParamsNames();
-                if (log.isDebugEnabled()) {
```
[the remark this answers is not in the document]
Comment (thc202) [reply zap/src/main/java/org/zaproxy/zap/authentication/ScriptBasedAuthenticationMethodType.java:521]: This was not addressed, other log statement was changed instead.

File: zap/src/main/java/org/zaproxy/zap/authentication/ScriptBasedAuthenticationMethodType.java [hunk 2]
```diff
@@ -927,13 +914,10 @@ public void handleAction(JSONObject params) throws ApiException {
                     // are there
                     String[] requiredParams = s.getRequiredParamsNames();
                     String[] optionalParams = s.getOptionalParamsNames();
-                    if (log.isDebugEnabled()) {
```
[the remark this answers is not in the document]
Comment (thc202) [reply zap/src/main/java/org/zaproxy/zap/authentication/ScriptBasedAuthenticationMethodType.java:930]: Ditto.

File: zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java [hunk 3]
```diff
@@ -595,10 +594,9 @@ public void regenerateAntiCsrfToken(HttpMessage message, HttpMessageSender httpS
         if (tokenValue != null) {
             // Replace token value - only supported in the body right now
             log.debug(
```
[the remark this answers is not in the document]
Comment (thc202) [reply zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java]: It was not added the debug log check.

[the remark this answers is not in the document]
Comment (thc202) [reply zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java]: The other check added should be removed though.

[the remark this answers is not in the document]
Comment (kingthorin) [reply zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java]: You mean the deprecation condition?

[the remark this answers is not in the document]
Comment (thc202) [reply zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java]: Yes, that was not in the original code.

**Comment by kingthorin** [comment]
Finger crossed :grinning:

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (thc202) [reply zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java]: No worries.

**Review by psiinon — APPROVED** [review]

**Pull request merged**